src/roles/roles.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { CreateRoleDTO } from './dto/create-role.dto';
+import { Role } from './roles.model';
+
+@Injectable()
+export class RolesService {
+    constructor(@InjectModel(Role) private roleRepository: typeof Role) {}
+
+    async createRole(dto: CreateRoleDTO) {
+        const role = await this.roleRepository.create(dto);
```

Comment:
there is no point in this syntax. async function anyway returns Promise. 

please check https://jakearchibald.com/2017/await-vs-return-vs-return-await/